apps/pie-docs/test/visual/navigation.spec.js

```diff
@@ -1,46 +1,44 @@
+import PERCY_BREAKPOINTS from './percy-breakpoints';
+import { test } from '../playwright/fixtures';
+import { disableCookieBanner, percySnapshot } from '../playwright/playwright-helper';
 import expectedRoutesJson from '../snapshots/expected-routes.snapshot.json';
 
-describe('PIE - Page Visual Tests', async () => {
-    beforeEach(async () => {
-        await browser.url(browser.options.baseUrl);
+test.describe('PIE - Page Visual Tests', () => {
+    test.beforeEach(async ({ page, context, baseURL }) => {
+        await page.goto(baseURL);
+        await disableCookieBanner(page, context, true);
     });
 
     expectedRoutesJson.forEach((route) => {
-        it(`Should respond take a screenshot of the requested route: - ${route}`, async () => {
-            const url = `${browser.options.baseUrl}/${route}`;
+        test(`Should respond take a screenshot of the requested route: - ${route}`, async ({ page }) => {
```

Comment:
super nit but I don't think you need the word `respond` in this test?

Reply:
@xander-marjoram you missed one. Get good.

Reply:
I'm so proud.